code/game/objects/items/singularityhammer.dm

```diff
@@ -85,6 +85,9 @@
 	throwforce = 30
 	throw_range = 7
 	w_class = WEIGHT_CLASS_HUGE
+	var/tesla_power = 20000
```

Comment:
```suggestion
	var/zap_power= 20000
```

yes while technically you are using tesla_zap, it isn't from a tesla so it shouldn't reference one.